.circleci/config.yml

```diff
@@ -218,6 +222,7 @@ jobs:
     environment:
       NX_E2E_CI_CACHE_KEY: e2e-circleci-macos
       NX_DAEMON: 'true'
+      NX_PERF_LOGGING: 'true'
```

Comment:
Shouldn't this have the native hasher as well?

Reply:
Yes, you're right.